romm.wiki/Tinfoil-Integration.md

```diff
@@ -0,0 +1,62 @@
+This will help you configure Tinfoil integration for your switch to work with your RomM Library.
+
+- What in Tinfoil?
+- Prepare
+- Configure 
+- Additional
+- Enjoy!
+
+## What is [Tinfoil](https://tinfoil.io/)? 
+<img src="https://cdn2.steamgriddb.com/icon_thumb/1178cf1b6a47d41fc664b7d97e305840.png" alt="drawing" width="200"/>
+
+
+Glad you asked!
+
+Tinfoil is an applet that is installed directly to a "modded" Nintendo Switch. The application will let you connect to something called "shops". Shops are basically a hosted resource that will you stream and install certain formatted files directly to your Nintendo Switch!
+
+
+## Prepare
+
+Please note down the following in order to make this as smooth as possible, as well as some pre-reqs:
+
+* RomM Updated to [RomM 3.5.0](https://github.com/rommapp/romm/releases/tag/3.5.0)
+* Add `DISABLE_DOWNLOAD_ENDPOINT_AUTH=true` to your docker compose and restart the containers.
+* URL Used to access RomM
+	* This can either be http or https - The system will prefer local access though to avoid reverse proxy issues.
+* Feed URL: `/api/tinfoil/feed`
+* Authentication details used to login to the Main RomM Instance. 
```

Comment:
```suggestion
* RomM updated to at least [3.5.0](https://github.com/rommapp/romm/releases/tag/3.5.0)
* Add `DISABLE_DOWNLOAD_ENDPOINT_AUTH=true` to your environment variables and restart the container
* The URL you use to access RomM
	* This can either be http or https
	* The system will prefer local access though to avoid reverse proxy issues
* Feed URL: `/api/tinfoil/feed`
* The username and password you use to login to RomM
```